Makefile

```diff
@@ -8,21 +8,24 @@ DATE := $(shell date +%Y-%m-%dT%H:%M:%S%z)
 PROJECT_NAME := $(shell basename "$(PWD)")
 
 # Go related variables.
-GO_FILES ?= $$(find . -name '*.go' | grep -v vendor)
+GO_FILES := $(shell find . -type f -name '*.go' | grep -v vendor)
 GOBASE := $(shell pwd)
 GOBIN := $(GOBASE)/bin
 GOPKG := $(.)
 GOENVVARS := GOBIN=$(GOBIN)
-GOCMD := $(GOBASE)/cli/node
-GOPROOF := $(GOBASE)/test/proofserver/cli
+GOCMD := $(GOBASE)/cmd/heznode
+GOPROOF := $(GOBASE)/test/proofserver/cmd
 GOBINARY := heznode
+# Go 1.13+ do not require GOPATH to be set, but we use some binaries in $GOPATH/bin
+GOPATH ?= $(shell go env GOPATH)
 PACKR := $(GOPATH)/bin/packr2
+GOCILINT := $(GOPATH)/bin/golangci-lint
 
 # Project configs.
 MODE ?= sync
-CONFIG ?= $(GOBASE)/cli/node/cfg.buidler.toml
+CONFIG ?= $(GOBASE)/cmd/heznode/cfg.buidler.toml
 PGHOST ?= localhost
-PGPORT ?= 5432
+PGPORT ?= 4012
```

Comment:
why this port?

Reply:
why not 5432? because 5432 is used by postgresql by default.
I am facing an issue where `make test` breaks the node database.

why exactly 4012? we need some random number between 1025 and 65535, lol.
4010 is used for api in tests:
https://github.com/hermeznetwork/hermez-node/blob/cf1212106c0ff99879c7c7853385e302b9df72fa/api/api_test.go#L44
so I used 4011 for debugapi and 4012 for db.